test/test_output.py

```diff
@@ -18,7 +18,6 @@
 
 from fontTools.ttLib import TTFont
 
-
```

Comment:
removing this line is not necessary. Keeping two empty lines between module-scope classes or function definitions is recommended in PEP8.